portfolio-project-matching-app/backend/dao.js

```diff
@@ -199,6 +172,55 @@ const createAssociation = async (coll, id1, id2) => {
     }
 }
 
+const createNewLike = async (projectId, userId) => {
+    /*
+    DESCRIPTION:    creates new likes document for provided project id and
+                    user id.
+
+    INPUT:          project id and user id in string format
+
+    RETURN:         NA
+    */
+    // get document snapshots for invalid input handling
```

Comment:
I should be able to capture this work in the `createAssociation` function, I'll look into it later, but it works for the time being.